### ISSUE TITLE
Postgresql cluster cannot mount PV

### ISSUE THREAD

**Issue by jiandanshifu** [zalando/postgres-operator#919]
I completely cloned the main code branch of github.Then， according to the documentation, the postgresql cluster was deployed.However, I found that ,each pod generated a pvc after deploying the postgresql cluster.However, one PV can only correspond to one pvc.Therefore, in the created postgresql cluster, only the first pod is running normally, and the other pods are all mounted with PV errors.More details are as follows.

The pivotal content of the yaml of the first pod is as follows(The name of PVC is pgdata-acid-minimal-cluster-0,the name of pod also is  pgdata-acid-minimal-cluster-0. ):

{
  "kind": "Pod",
  "apiVersion": "v1",
  "metadata": {
    "name": "acid-minimal-cluster-0",
    "generateName": "acid-minimal-cluster-",
    "namespace": "default",
    "labels": {
      "application": "spilo",
      "controller-revision-hash": "acid-minimal-cluster-7d6b948894",
      "kubernetes": "acid-minimal-cluster",
      "statefulset.kubernetes.io/pod-name": "acid-minimal-cluster-0",
      "team": "acid"
    },
    "annotations": {
      "cni.projectcalico.org/podIP": "172.17.5.16/32",
    "ownerReferences": [
      {
        "apiVersion": "apps/v1beta1",
        "kind": "StatefulSet",
        "name": "acid-minimal-cluster",
        "uid": "b591b418-7ef9-11ea-8edc-0050568b9cea",
        "controller": true,
        "blockOwnerDeletion": true
      }
    ]
  },
  "spec": {
    "volumes": [
      {
        "name": "pgdata",
        "persistentVolumeClaim": {
          "claimName": "pgdata-acid-minimal-cluster-0"
        }
      },
      {
        "name": "dshm",
        "emptyDir": {
          "medium": "Memory"
        }
      },
      {
        "name": "postgres-pod-token-czc46",
        "secret": {
          "secretName": "postgres-pod-token-czc46",
          "defaultMode": 420
        }
      }
    ],
    "containers": [
      {
        "name": "postgres",
        "image": "registry.opensource.zalan.do/acid/spilo-12:1.6-p2",
        "volumeMounts": [
          {
            "name": "pgdata",
            "mountPath": "/home/postgres/pgdata"
          }
        ], 
        }
      }
    ],
  },
}

The pivotal content of the yaml of the second pod is as follows(The name of PVC is pgdata-acid-minimal-cluster-1,the name of pod also is  pgdata-acid-minimal-cluster-1. ):

{
  "kind": "Pod",
  "apiVersion": "v1",
  "metadata": {
    "name": "acid-minimal-cluster-1",
    "generateName": "acid-minimal-cluster-",
    "namespace": "default",
    "selfLink": "/api/v1/namespaces/default/pods/acid-minimal-cluster-1",
    "uid": "b718180d-7ef9-11ea-8edc-0050568b9cea",
    "resourceVersion": "4919040",
    "creationTimestamp": "2020-04-15T09:15:57Z",
    "labels": {
      "application": "spilo",
      "controller-revision-hash": "acid-minimal-cluster-7d6b948894",
      "kubernetes": "acid-minimal-cluster",
      "statefulset.kubernetes.io/pod-name": "acid-minimal-cluster-1",
      "team": "acid"
    },
    "ownerReferences": [
      {
        "apiVersion": "apps/v1beta1",
        "kind": "StatefulSet",
        "name": "acid-minimal-cluster",
        "uid": "b591b418-7ef9-11ea-8edc-0050568b9cea",
        "controller": true,
        "blockOwnerDeletion": true
      }
    ]
  },
  "spec": {
    "volumes": [
      {
        "name": "pgdata",
        "persistentVolumeClaim": {
          "claimName": "pgdata-acid-minimal-cluster-1"
        }
      },
      {
        "name": "dshm",
        "emptyDir": {
          "medium": "Memory"
        }
      },
      {
        "name": "postgres-pod-token-czc46",
        "secret": {
          "secretName": "postgres-pod-token-czc46",
          "defaultMode": 420
        }
      }
    ],
    "containers": [
      {
        "name": "postgres",
        "image": "registry.opensource.zalan.do/acid/spilo-12:1.6-p2",
        "volumeMounts": [
          {
            "name": "pgdata",
            "mountPath": "/home/postgres/pgdata"
          },
          {
            "name": "dshm",
            "mountPath": "/dev/shm"
          },
          {
            "name": "postgres-pod-token-czc46",
            "readOnly": true,
            "mountPath": "/var/run/secrets/kubernetes.io/serviceaccount"
          }
        ],
      }
    ],
    "restartPolicy": "Always",
    "terminationGracePeriodSeconds": 300,
    "dnsPolicy": "ClusterFirst",
    "serviceAccountName": "postgres-pod",
    "serviceAccount": "postgres-pod",
    "nodeName": "192.168.56.109",
  },
  "status": {
    "phase": "Pending",
    "conditions": [
      {
        "type": "Initialized",
        "status": "True",
        "lastProbeTime": null,
        "lastTransitionTime": "2020-04-15T09:15:57Z"
      },
      {
        "type": "Ready",
        "status": "False",
        "lastProbeTime": null,
        "lastTransitionTime": "2020-04-15T09:15:57Z",
        "reason": "ContainersNotReady",
        "message": "containers with unready status: [postgres]"
      },
      {
        "type": "PodScheduled",
        "status": "True",
        "lastProbeTime": null,
        "lastTransitionTime": "2020-04-15T09:15:57Z"
      }
    ],
    "hostIP": "192.168.56.109",
    "startTime": "2020-04-15T09:15:57Z",
    "containerStatuses": [
      {
        "name": "postgres",
        "state": {
          "waiting": {
            "reason": "ContainerCreating"
          }
        },
        "lastState": {},
        "ready": false,
        "restartCount": 0,
        "image": "registry.opensource.zalan.do/acid/spilo-12:1.6-p2",
        "imageID": ""
      }
    ],
  }
}

In short, each pod of the postgresql cluster has created its own PVC, but the kubernetes cluster only provides a PV for the postgresql cluster. For the first time, please give pointers!
